extensions/qute/deployment/pom.xml

```diff
@@ -25,6 +25,11 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-arc-deployment</artifactId>
         </dependency>
+        <!-- Panache SPI-like module to collect entity classes -->
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-panache-common-deployment</artifactId>
```

Comment:
Can this extension be marked optional and be included only when Panache is used with Qute somehow?

Reply:
It's an SPI module, it doesn't depend on anything.

Reply:
This extension should not contain any logic. TBH I'm not sure what happens if I add `<optional>true</optional>`...

Reply:
@aloubyansky Does `<optional>true</optional>` in a deployment module POM affect the quarkus build in any way?

Reply:
@mkouba optional dependencies of an extension will not be added to the application build classpath

Reply:
In this case, the dependency is an SPI-like `-deployment` module and we use a class from this module in a build step, i.e. the class needs to be added on the app build class path. So I think it's ok not to add `<optional>true</optional>` here, right?

Reply:
That sounds like it doesn't qualify as optional at all.